provider-tutorials/becoming-erigon-provider.md

```diff
@@ -0,0 +1,119 @@
+# Tutorial for becoming Erigolem provider
+
+:information_source: `Erigolem` is a codename for "Erigon on Golem" project. More information can be found at 
+[Example service - managed Erigon](https://github.com/golemfactory/yagna-docs/tree/master/requestor-tutorials/service-development/service-example-2-managed-erigon.md)
+
+Here we describe an [installation script](https://github.com/golemfactory/yagna-service-erigon/blob/master/install_provider.sh) which should make the whole process of becomming Erigolem provider fun and smooth. Script is prepared for Ubuntu 18.04 or later and should work on other Linux distributions after some modifications.
+It's not intended to be run on personal computers because
+
+* it creates separate user on which provider daemon will be run
+* it installs nginx web server and certbot
+* optionally it will generate Let's Encrypt certificate and set certbot to remind you about certificate renewals
+
+## Requirements
+
+* Ubuntu 18.04 or later
+* Root privileges
+* Public IP address or DNS name
+* Ability to open a port for the Erigon service
+
+By default nginx reverse-proxy is listening on standard Erigon port `8545`. You need to open this port for ingres traffic.
+If you wish your Erigolem service to be available on different port number (e.g. `8454` cannot be opened) you need to edit the installation script.
+
+<details>
```

Comment:
I'm not sure if this renders correctly in GitBook. Have you checked it?

Reply:
No I haven't it's the most fancy trick I do in markdown. I'll ask someone with experience for the advice.

Reply:
@shadeofblue adviced to merge this branch to develop first (which is generally a recommended practice)